version.go

```diff
@@ -14,4 +14,4 @@
 
 package lksdk
 
-const Version = "2.12.1"
+const Version = "2.12.2"
```

Comment:
I don't think this is urgent that we need to cut a new release.

Reply:
if we want it there soon, we should bump. it's not breaking as far as cloud-agents is concerned anyway